chart-modules/common/picasso/chart-builder/presets/dimension-measure-chart/layout-settings.js

```diff
@@ -20,9 +20,11 @@ export default {
   },
 
   refLines: {
-    minimumLayoutMode: 'SPARK',
-    prioOrder: 60,
-    displayOrder: -10,
+    layout: {
+      minimumLayoutMode: 'SPARK',
+      prioOrder: 60,
+      displayOrder: 10000,
```

Comment:
would a slightly lower value work?
I would like it to be smaller then the displayOrder for selections components (ex: lasso + range select)

Reply:
To be honest I don't know what value range is supposed to be used. So what number do you suggest?

Reply:
I recommend a new number (not used by anything else)
smaller then 10000 and larger then anything that it should render over (probably >1100 if it should render over points in  dist-plot)

Reply:
Ok, will try and update.